testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_replication_eventing.py

```diff
@@ -484,3 +485,75 @@ def test_replication_delete_event(params_from_base_test_setup, num_of_docs):
     doc_ids = db.getDocIds(cbl_db)
     for sg_doc in docs_to_modify:
         assert sg_doc["id"] not in doc_ids, "channel access removal didn't purge the docs from cbl db"
+
+
+@pytest.mark.listener
+@pytest.mark.replication
+@pytest.mark.parametrize("attachment_generator, attachment_file_list", [
+    [None, None],
+    [load_from_data_dir, ["Sample_Image_30mb.jpg"]],
+    [generate_2_png_100_100, None],
+])
+def test_push_replication_for_20mb_doc(params_from_base_test_setup, attachment_generator, attachment_file_list):
+    """
+    @summary:
+    1. Create docs in CBL and replicate to SG using push one-shot replication
+    2. Add update to SG and CBL to create conflict
```

Comment:
I do not see this step implemented. If this step not required, can you remove it from summary?

Reply:
removed.